tscached/readahead.py

```diff
@@ -9,7 +9,7 @@
 from tscached.shadow import perform_readahead
 
 
-if __name__ == '__main__':
+def start():
     parser = argparse.ArgumentParser(description='Harness for querying/testing tscached/kairosdb')
```

Comment:
Probably doesn't matter but does this description fit the script? Since this is the readahead that should be triggered via cron or whatnot right?